tests/e2e/dsc_creation_test.go

```diff
@@ -119,11 +119,8 @@ func (tc *testContext) testAllApplicationCreation(t *testing.T) error {
 			if err != nil {
 				require.NoError(t, err, "error validating application %v when enabled", tc.testDsc.Spec.Components.Dashboard.GetComponentName())
 			}
-		} else {
-			if err == nil {
-				require.NoError(t, err, "error validating application %v when disabled", tc.testDsc.Spec.Components.Dashboard.GetComponentName())
-			}
```

Comment:
🤔 
this likely was a typo even in the original/previous writing?

Reply:
Yeah, this test was a mess. I've cleaned it up.